multiversx_sdk/core/transaction.py

```diff
@@ -1,22 +1,10 @@
-from typing import Optional, Protocol
+from typing import Optional
 
-from multiversx_sdk.core.constants import (DEFAULT_HRP,
-                                           TRANSACTION_MIN_GAS_PRICE,
+from multiversx_sdk.core.constants import (TRANSACTION_MIN_GAS_PRICE,
```

Comment:
So the `IAddressConverter` was never used? Though, we can mention this removal as a breaking change, perhaps, as well.

Reply:
It was not used. Added in the PR description.